File: number_of_words.java
import java.util.*;
public class number_of_words
{
public static void main(String ar[])throws Exception
{
Scanner sc=new Scanner(System.in);
String p=new String();
p=sc.next();
String q[]=p.split("\\s");
System.out.println("number of words in a given line is:"+q.length);
}
}

File: number_of_words.py
n=[str(k) for k in input().split()]
m=int(input())
dict={n[i]:len(n[i]) for i in range(0,len(n))}
p=0
for i in range(len(n)):
    if(len(n[i])!=m):
        p=p+1
print("number of elements:",p)  
